td-agent/templates/usr/lib/tmpfiles.d/td-agent.conf

```diff
@@ -1,4 +1,5 @@
 d /tmp/<%= project_name %> 0755 <%= project_name %> <%= project_name %> - -
+d /var/run/<%= project_name %> 0755 <%= project_name %> <%= project_name %> - -
```

Comment:
Nit: /var/run should be mounted as tmpfs. Shouldn't we preserve /var/run contents for td-agent?

Reply:
/var/run/td-agent/ is used to store td-agent.pid only, so I thought it is ok.

what "contents" means?

Reply:
> what "contents" means?

It means some files are put there.